README.md

```diff
@@ -35,3 +35,12 @@ Make sure the service is up first using `docker-compose up` then run:
 ```sh
 docker-compose exec core python manage.py test
 ```
+
+to run unit tests. Or, to load cancer-static data:
+
+```
+docker-compose exec core bash
+python manage.py acquiredata 
```

Comment:
Is there a reason to not do `acquire_data` and `load_data`? Readability would be improved.